tests/test_metrics.py

```diff
@@ -86,4 +89,8 @@ def test_metrics(farm, runner):
         assert count_forward[ind].value == N_CALLS
         assert farm.counters()[p][ENDPT_NAME_FWD] == N_CALLS
 
+    # Expect only queue wait time for "status", because that's what `/metrics` gets routed to
+    assert len(count_wait_time) == 1
+    assert count_wait_time["status"].value == 2
```

Comment:
Why is this `2`? It waited in the queue for 2 seconds?

Reply:
Two observations in that bucket.